db/migrate/20190612144842_add_user_attributes.rb

```diff
@@ -1,8 +1,8 @@
 class AddUserAttributes < ActiveRecord::Migration[5.2]
   def change
-    add_column          :users, :gender, :integer 
-    change_column_null  :users, :gender, false   
+    add_column          :users, :gender, :integer
```

Comment:
You can use null: false and avoid the next line:
add_column          :users, :gender, :integer, null: false

Same for the next cases

Reply:
Yes, thats makes sense. But sqlite complains when trying to add a column with not null constraint and no default value.

http://www.rhuberdeau.com/articles/18-Rails-SQLite3-SQLException-Cannot-add-a-NOT-NULL-column-with-default-value-NULL

Reply:
Ok got it 👍

Reply:
We are migrating to Postgresql in the next PR